service/src/main/java/io/camunda/service/transformers/filter/IncidentFilterTransformer.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.transformers.filter;
+
+import static io.camunda.search.clients.query.SearchQueryBuilders.and;
+import static io.camunda.search.clients.query.SearchQueryBuilders.longTerms;
+import static io.camunda.search.clients.query.SearchQueryBuilders.stringTerms;
+import static io.camunda.search.clients.query.SearchQueryBuilders.term;
+
+import io.camunda.search.clients.query.SearchQuery;
+import io.camunda.service.search.filter.DateValueFilter;
+import io.camunda.service.search.filter.IncidentFilter;
+import io.camunda.service.transformers.ServiceTransformers;
+import io.camunda.service.transformers.filter.DateValueFilterTransformer.DateFieldFilter;
+import java.util.List;
+
+public class IncidentFilterTransformer implements FilterTransformer<IncidentFilter> {
+
+  private final ServiceTransformers transformers;
```

Comment:
💭  (minor) Should we rename this variable to `serviceTransformers` or `transformerServices`?